addons/mail/static/tests/web/chatter_tests.js

```diff
@@ -315,7 +315,6 @@ QUnit.test("base rendering when chatter has no record", async () => {
     await contains(".o-mail-Message-body", { text: "Creating a new record..." });
     await contains("button", { count: 0, text: "Load More" });
     await contains(".o-mail-Message-actions");
-    await contains(".o-mail-Message-actions i", { count: 0 });
```

Comment:
This test is no longer relevant since the translation are enabled by default on the mocked test setup and therefore the "Translate" action is available. I could also rewrite this test to specificaly assert that the action mentionned before is displayed but that would be counter productive as this is the responsibility of "translation_test.js" file.